### DISCUSSION TITLE
Aqara E1 Door and Window Sensors (MCCGQ14LM) not working with Zigbee2Tasmota 10.1.0

### DISCUSSION THREAD

**Discussion by namirda** [arendst/Tasmota#14543] [Ideas]
The latest E1 Door and Window Sensors from Aqara do not work with Zigbee2Tasmota v 10.1.0.

After several attempts and button clicking, the devices can be made to pair but do not provide Contact or Battery information and do not respond to the magnet.

Pressing the button on the devices gives the following on the console :

23:52:56.283 MQT: tele/tasmota_zigbee/SENSOR = {"ZbReceived":{"0xF39E":{"Device":"0xF39E","ModelId":"lumi.magnet.acn001","Endpoint":1,"LinkQuality":60}}}

This problem has been reported in other Zigbee environments such as Zigbee2MQTT, Deconz and ZHA - see these threads

[https://github.com/dresden-elektronik/deconz-rest-plugin/issues/5403](https://github.com/dresden-elektronik/deconz-rest-plugin/issues/5403)
[https://github.com/Koenkk/zigbee2mqtt/issues/9274](https://github.com/Koenkk/zigbee2mqtt/issues/9274)
[https://github.com/home-assistant/core/issues/58985](https://github.com/home-assistant/core/issues/58985)

It seems that these new Aqara devices do not strictly conform to Zigbee standards but despite that have been made to work in these other Zigbee environments.

Any chance of them being supported in Z2T?

Thanks

**Comment by s-hadinger** [comment]
It really depends on how far from the standard the device is. It may require changes in the engine, or could be handled by a rule.

Those are long threads, any chance there is a summary of the new attributes?

**Comment by namirda** [comment]
Indeed they are long threads. The issue seems to be that this version of the sensor expects to receive the Aqara manufacturer number (0x115f) as a response back during pairing. 

I think a change to the engine is going to be required to fix this.

Z2T apparently uses the EmberZNet stack - how are updates arranged/coordinated with Silabs?

Z2T currently uses v6.7.9 - there are later versions available but what is required to make them work in Z2T?

Sorry - I really don't know how this stuff works!

Thanks

**Comment by darek-margas** [comment]
Is there any progress on such compatibility?
